tests/postgres_tests/test_array.py

```diff
@@ -424,6 +424,32 @@ class MyModel(PostgreSQLModel):
         self.assertEqual(len(errors), 1)
         self.assertEqual(errors[0].id, 'postgres.E002')
 
+    def test_invalid_default(self):
+        class MyModel(PostgreSQLModel):
+            field = ArrayField(models.IntegerField(), default=[])
+
+        model = MyModel()
+        errors = model.check()
+        self.assertEqual(len(errors), 1)
+        self.assertEqual(errors[0].id, 'postgres.E003')
```

Comment:
It should be fine to check the complete warning similar to: https://github.com/django/django/blob/4d60261b2a77460b4c127c3d832518b95e11a0ac/tests/check_framework/test_model_field_deprecation.py#L17-L23 I forget if there was a reason the existing test doens't use that approach.

Reply:
Should I change the existing tests too? `test_field_checks`, `test_invalid_base_fields`, `test_nested_field_checks`

Reply:
No, at least not part of this PR.